pages/en/lb2/Deploy-for-GDPR-readiness.md

```diff
@@ -0,0 +1,212 @@
+such as database connection information (IP addresses and credentials), in order to connect to the backend resources:
+    - `server/config.json`
+    - `server/datasources*.json`
+    - `server/datasources*.js`
+    - `server/model-config.json`
+    - `server/middleware*.json`
+
+2. Application data
+
+    LoopBack provides built-in User model that makes the LoopBack applications require authentication. However, it is not compulsary to use.  Besides, developers can create other models that involve personal data, which the LoopBack framework has no control over.  
+
+It is developer's responsibility to design their LoopBack application using the [GDPR Privacy By Design](https://www.eugdpr.org/more-resources-1.html) as guidelines.
+
+#### How is the data being collected
+
+1. Application configuration data
+
+    All information captured in the application configuration files are provided by the developers of the LoopBack application during development time. 
+
+2. Application data
+
+    During the runtime of the application, if the LoopBack application requires authentication, the userid and password might be involved.  Depending on the application developed by the LoopBack users, it might ask the end-users for other personal data. 
+
+#### When is the data being collected?
+LoopBack is a development framework that does not collect any data and send it back to IBM.  However, there might be personal data involved in a LoopBack application developed by the users. 
+
+1. Application configuration data
+
+    All information captured in the application configuration files are provided by the developers of the LoopBack application during development time. 
+
+2. Application data
+
+    Depending on the LoopBack application developed by the LoopBack users, application data might be collected at different times during runtime of the application.
+
+##### Personal data used for online contact with IBM
+
+Loopback clients can submit online comments/feedback/requests to contact IBM about LoopBack subjects.
+
+Regarding getting support from IBM as open source community users or paid customer support, LoopBack users might be asked to provide a sample application and output/debug logs for problem reproduction and problem determination. It is important to note that personal data is not needed for this purpose.  If there is personal data included in the application, they should be removed or deidentified before sending it to IBM Support.
+
+For supporting the community users through GitHub, please keep in mind that it is a public platform and anyone who has access to World Wide Web is able to view it. If there is personal data included in the application, they should be removed or deidentified before sending it to IBM Support.
+
+Typically, only the client name and email address are used, to enable personal replies for the subject of the contact, and the use of personal data conforms to the [IBM Online Privacy Statement] (https://www.ibm.com/privacy/us/en/).
```

Comment:
`... are used, in order to enable personal replies for the subject of the contact. The use of personal data conforms ...`